.circleci/config.yml

```diff
@@ -1,91 +1,91 @@
 version: 2
 jobs:
-  install-job:
```

Comment:
Prettier changed all the whitespace here, so disable whitespace in the Github diff settings in order to see the meaningful changes (which is node version updated and npm->yarn)